nest.js

```diff
@@ -0,0 +1,16 @@
+function Nest(x, y, d, theta) {
+  this.pos = createVector(x, y);
+  this.d = d;
```

Comment:
Do these variables have to have single-letter variable names from p5.js? It is difficult for someone reading through to know what they reference.